client/starwhale/base/uri/project.py

```diff
@@ -40,6 +54,13 @@ def __init__(
             raise Exception("can not init project with empty name")
         self.name = name
         # TODO check if project exists for local and remote
+        if self.instance.is_cloud:
+            # TODO check whether contains namespace in name(like 'sw:project')?
+            self.name = (
+                int(self.name)
+                if self.name.isdigit()
```

Comment:
when the type of name is int, `self.name.isdigit()` will raise Exception?

Reply:
fixed, unify to use str